### PR TITLE
VR: Recreation and training programs

### PR DESCRIPTION
**Pull request by Cdey78**
Posting this for review, criticism,  ect.

Implements the VR system we have!
Engineering training VR: simply put, it's a VR world with a hugbox supermatter. For educational purposes.
Recreational VR: contains a kitchen,  dining room, general relaxation areas, thunderome arena, bathroom, art supplies, and a hydroponics.

:cl: Cdey
add: VR now works!
add: Two types of VR worlds: Training, and recreation.
add: This means that you can teach people how to run the supermatter with 100% safety!
experiment: Please report any bugs you find with this, as I'm sure there are plenty of them.
/:cl:
Engineer education world:
image

Recreation deck:
image

image

### CODE REVIEW

File: _maps/basemap.dm [hunk 1]
```diff
@@ -20,6 +21,7 @@
 //#include "map_files\generic\Space.dmm"
 //#include "map_files\generic\SpaceDock.dmm"
 #include "map_files\generic\Transit.dmm"
+//#include "map_files\generic\VR_Space.dmm"
```
Comment (ninjanomnom) [inline _maps/basemap.dm]: Why are you adding commented out map files?

Comment (Cdey78) [reply _maps/basemap.dm]: That was me attempting to make them their own Z-Level, but that's not really required.

**Comment by Cdey78** [comment]
They're gone now.

**Comment by RemieRichards** [comment]
Someone finally using my VR system, 👍

**Comment by praisenarsie** [comment]
you should port this for tg 😉😉😉😉😉😉😉😉😉😉😉😉😉😉

**Comment by davethwave** [comment]
Also perhaps in a later pr have a Atmos version of it since only 1/10 people only know how to do it and only 1/30 people know how to do it well.

**Comment by Vivalas** [comment]
Still needs mapmerging.

**Comment by Kmc2000** [comment]
Still needs mapmerging.

**Comment by crossedfall** [comment]
EoL

**Pull request closed without merging**